contents/careers/designer.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Designer
+sidebar: Careers
+showTitle: true
+---
+
+"Design is not just what it looks like and feels like. Design is how it works." - Steve Jobs, co-founder of [Apple](https://apple.com).
+
+PostHog exists to increase the number of successful products in the world.
+
+We're not just building software, our product is the company. This role is to shape how we get design into every part of that - from brand, to web presence, our engagement with our blossoming community, and into the platform itself.
+
+## Ambition and engineering have taken us a really long way, so far
+
+PostHog's engineering culture is best-in-class, and we've had explosive user growth from this alone.
+
+We [launched a four week old minimum viable product in February](/handbook/company/story), and since then have been deployed in thousands of places around the world.
+
+We have are used to help indie game designers make it more fun to defend earth from an alien threat. We are used by mulitnational organizations running software that powers everything from banks to airlines. We are used by startups disrupting their own industries.
+
+We're a company like no other in our space. Our approach is bottoms up and that starts by being great for developers to install and use.
+
+By being open source, we can be used on any software project throughout the world, for free, forever. Some developers will use the platform on a side project, others at their startup or small business, and others in their Fortune 500. We are building a true platform that can grow from 1 user to many no matter the size or shape of the organization.
+
+The core of our approach is to delight end users. It's not about executive dashboards and then a terrible interface for everyone else. It's the sense of power we give to the person on the ground, doing the actual work, every day.
+
+## But, there's a catch.
+
+We aren't on track to achieve this, as we are missing *you*.
+
+You will be responsible for getting world class design into every part of our company. We'll team you up with an incredible graphic designer, and with our developer advocate and engineering team to create something special. You'd be one of the very first design hires we'd make at a pivotal stage for the company.
+
+## This role isn't scoped tightly
+
+The reason we hire exceptional people is so they can tell us what to do.
+
+Success in this role means it's obvious our website, docs, product and presence are world class. We are focussed on increasing user acquisition, retention and our community growth. Design is core to impacting all of these things.
+
+## Does this sound like you?
```

Comment:
This ad might be missing "benefits" or "what it's like to work at posthog" style things, but maybe that's on purpose?

Reply:
it felt kind of silly listing a bunch of bullet points, but I should make an overall benefits list and I should link to this more clearly

Reply:
Fixed